src/NLog.Targets.Seq/SeqTarget.cs

```diff
@@ -51,6 +52,11 @@ public SeqTarget()
         public string ApiKey { get; set; }
 
         /// <summary>
+        /// The address of the proxy to use, including port separated by a colon. If not provided, default operating system proxy will be used.
+        /// </summary>
+        public string Proxy { get; set; }
```

Comment:
Since this is only the `address` component of the `WebProxy`, perhaps `ProxyAddress` would be a better name for the property?